### DISCUSSION TITLE
求助：Mpv外接Svp用RIFE AI engine无法补帧

### DISCUSSION THREAD

**Discussion by mmnmuo** [hooke007/mpv_PlayKit#486] [libmpv]
安装了svp 4 Pro官网上下的版本，以及mpv.net_cm
mpv外接svp补帧的时候，选用其他的都行，唯独开不了rife ai引擎
报的错我也没有代码基础看不懂，求个大佬解释一下
![Desktop Screenshot 2024 07 21 - 16 44 19 22](https://github.com/user-attachments/assets/fd59c97f-b9e4-4b85-9f6f-c4b9e3da9134)
![Desktop Screenshot 2024 07 21 - 16 46 13 01](https://github.com/user-attachments/assets/a49af0f7-1d5f-410a-b64d-0aa03df97a93)

**Comment by hooke007** [comment]
svp自己生成的脚本报错了，它自己的环境没检测到对应的函数

**Reply by mmnmuo** [reply]
那我有什么解决方法吗？比如说我能进到他的环境吗？
我用potplayer利用他官网上的信息可以使用rife。
而且我貌似记得我之前也用mpvnet_cm使用成功过rife，而且作为一个小白，我还下载了mpv lazy，后来删掉之后就无法成功使用了。
报错中的“src\cython\vapoursyn.pyx”我没找到，我猜是在他环境里的？
求解答

**Reply by hooke007** [reply]
`src\cython\vapoursyn.pyx` 是报错信息定义的地方，和事实错误无关

**Reply by mmnmuo** [reply]
又自己整了一整天，还Python学了点入门，实在没办法了(T_T)

**Reply by hooke007** [reply]
我不研究它，建议直接用它自带的播放器

**Reply by mmnmuo** [reply]
它自带的播放器？是指它指定的播放器吗？

**Reply by hooke007** [reply]
它的安装步骤里就有装mpv的选项，这就是它自带的播放器
![image](https://github.com/user-attachments/assets/a031595a-0ed9-45c6-a345-bee1ad236d5e)
